src/UI/View.hs

```diff
@@ -31,19 +33,15 @@ renderView = cata $ \ view -> wrapR . Inset (Size 5 3) $ case view of
     maxSize <- liftFR GetMaxSize
     liftFL (Draw.Text maxSize s)
   Label s -> liftFL (Draw.Text (pure Nothing) s)
-  List children -> fromMaybe (pure 0) (foldr stack Nothing (intersperse spacer children))
+  List children -> maybe (pure 0) sconcat (nonEmpty (intersperse spacer children))
```

Comment:
This is a pretty great improvement; the `stack` definition in the `where` clause has been 🔥’d.